frontend/src/metabase/public/components/EmbedFrame/EmbedFrame.tsx

```diff
@@ -158,7 +158,7 @@ export const EmbedFrame = ({
     : [];
   const hasVisibleParameters = visibleParameters.length > 0;
 
-  const hasHeader = Boolean(finalName || dashboardTabs);
+  const hasHeader = Boolean(finalName || dashboardTabs || downloadsEnabled);
```

Comment:
This fixes the bug of the header not showing up when titled=false and there's only one tab

Reply:
nit: I wonder if `downloadsEnabled` should be outside of the `Boolean(...)` cast, as it is already a boolean.